runtime/src/iree/base/config.h

```diff
@@ -185,6 +185,30 @@ typedef IREE_DEVICE_SIZE_T iree_device_size_t;
 #define IREE_TRACING_MODE 0
 #endif  // !IREE_TRACING_MODE
 
+//===----------------------------------------------------------------------===//
+// Allocator configuration
+//===----------------------------------------------------------------------===//
+
+#if !defined(IREE_ALLOCATOR_ENABLE_MIALLOC)
+// Default disable MIALLOC in certain conditions:
```

Comment:
It's "mim" "alloc", not "mi" "alloc", right? These comments and `#define` values look like they have a typo (missing an `M`)